recipe/meta.yaml

```
@@ -81,6 +85,7 @@ test:
   requires:
     - git
     - pocl      # [linux]
+    - cuda-driver-dev
```

Think we want to limit this to CUDA 12

```suggestion
    - cuda-driver-dev  # [(linux64 or ppc64le or win) and (cuda_compiler_version or "").startswith("12")]
```

---

Lets give it a shot